src/AWS.Deploy.CLI/AWS.Deploy.CLI.csproj

```diff
@@ -21,11 +21,11 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="AWSSDK.IdentityManagement" Version="3.7.200.42" />
-    <PackageReference Include="AWSSDK.CloudFormation" Version="3.7.203.36" />
-    <PackageReference Include="AWSSDK.SecurityToken" Version="3.7.201.38" />
-    <PackageReference Include="AWSSDK.SSO" Version="3.7.200.42" />
-    <PackageReference Include="AWSSDK.SSOOIDC" Version="3.7.200.42" />
+    <PackageReference Include="AWSSDK.IdentityManagement" Version="3.7.301.3" />
```

Comment:
Can you update the internal tracker and use that to regenerate https://github.com/aws/aws-dotnet-deploy/blob/main/THIRD_PARTY_LICENSES?

Reply:
Done

Reply:
Did you update the internal tool? I still see AWSSDK.IdentityManagement at 3.7.200.42?

Reply:
Just updated the internal tool as well.